chainer/optimizer.py

```diff
@@ -384,8 +384,12 @@ def update(self, lossfun=None, *args, **kwds):
 
         """
         if lossfun is not None:
+            use_cleargrads = kwds.pop('use_cleargrads', False)
```

Comment:
If the name `use_cleargrads` is already used, this PR breaks backward compatibility. I think it is too pessimistic to consider such case.